webpack.config.renderer.js

```diff
@@ -175,10 +173,7 @@ module.exports = merge(base, {
         __filename: false,
         __dirname: false,
     },
-    target: 'electron-renderer',
-    devServer: {
-        port: WEBSERVER_PORT,
-    },
+    target: 'web',
```

Comment:
For my own curiosity, what does changing the target to web do? Is the old one removed or is there some other reason you changed it?

Reply:
I believe if you use `electron-renderer` it assumes that `nodeIntegration` was enabled meaning that `require` should be exposed. This isn't usually a problem, but with chunking multiple files it is because it will use NodeJS `require` instead of the webpack baked-in function.

It's probably safer altogether to compile this for web, since we should assume anything run in the renderer process only has access to the same global fields that a Chrome process would.